doc/src/devdocs/llvm.md

```diff
@@ -112,36 +112,36 @@ The last step is labor intensive.  Suggestions on a better way would be apprecia
 
 Julia has a generic calling convention for unoptimized code, which looks somewhat
 as follows:
-```
+```c
     jl_value_t *any_unoptimized_call(jl_value_t *, jl_value_t **, int);
```

Comment:
I believe they end up showing as indented in that case. Dunno whether that was intentional.